### PR TITLE
Exclude some examples from docs navigation

### PR DESCRIPTION
**Pull request by awaelchli**
## What does this PR do?

We are updating the course notebooks (https://github.com/Lightning-AI/tutorials/pull/232) as well as the rest of the examples (https://github.com/Lightning-AI/tutorials/pull/233) to Lightning 2.0. However, some examples turned out to be problematic, and we decided to temporarily exclude them until they are updated. The goal is to release 2.0 with docs that have valid tutorials. 

cc @borda

### CODE REVIEW

File: docs/source-pytorch/conf.py [hunk 1]
```diff
@@ -49,7 +49,37 @@
...
 if _SHOULD_COPY_NOTEBOOKS:
     AssistantCLI.copy_notebooks(
-        PATH_RAW_NB, PATH_HERE, "notebooks", patterns=[".", "course_UvA-DL", "lightning_examples"]
+        PATH_RAW_NB,
+        PATH_HERE,
+        "notebooks",
+        # TODO: Complete converting the missing items and add them back
+        patterns=[
+            ".",
+            "course_UvA-DL/01-introduction-to-pytorch",
```
Comment (Borda) [inline docs/source-pytorch/conf.py:58]: this is cool :heart:

**Review by Borda — APPROVED** [review]

File: docs/source-pytorch/conf.py [hunk 2]
```diff
@@ -49,8 +49,24 @@
...
+        "lightning_examples/augmentation_kornia",
+        "lightning_examples/finetuning-scheduler",
+        "lightning_examples/reinforce-learning-DQN",
+        "lightning_examples/text-transformers",
+        "lightning_examples/warp-drive",
+    ]
+    for file in ignore:
+        file = os.path.join(PATH_HERE, "notebooks", file)
+        if os.path.exists(file):
+            os.remove(file)
```
Comment (carmocca) [inline docs/source-pytorch/conf.py:69]: Is this going to create undesired local changes when creating the docs?

**Review by carmocca — APPROVED** [review]

File: docs/source-pytorch/conf.py (hunk 2 above)
Comment (awaelchli) [reply docs/source-pytorch/conf.py:69]: Nope, the notebooks folder referenced here was always generated. It is basically the same type of artifact as the build folder for the docs. whatever we add or remove here won't be seen by the developer

**Review by Borda — CHANGES_REQUESTED** [review]
let's wait for the commit when all updated notebooks are build
https://dev.azure.com/Lightning-AI/Tutorials/_build/results?buildId=144811&view=results

**Review by Borda — APPROVED** [review]

**Pull request merged**